_sass/modules/_buttons.scss

```diff
@@ -148,3 +149,48 @@
     box-shadow: 0 5px 20px 0 rgba($footerColor, 0.28);
   }
 }
+
+.btn-bordered {
+  font-size: 11px;
+  font-weight: 500;
+  color: $textColor;
+  border: 1px solid rgba($black, .2);
+  background-color: transparent;
+  padding: 9px 16px;
+  border-radius: 2px;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  -webkit-transition: all 0.34s ease-in-out;
+  -moz-transition: all 0.34s ease-in-out;
+  -o-transition: all 0.34s ease-in-out;
+  transition: all 0.34s ease-in-out;
+
+  &:hover, &:focus {
+    color: $mainBrandColor;
+    background-color: rgba($mainBrandColor, .08);
+    border: 1px solid rgba($mainBrandColor, .3);
+  }
+
+  &.with-icon {
+    padding: 9px 16px 9px 10px;
+    &:before {
+      content: '';
+      display: inline-block;
+      background: url('../img/icon-arrow-back.svg') no-repeat center/cover;
+      width: $icon-size--xs;
+      height: $icon-size--xs;
+      vertical-align: middle;
+      margin: 0 8px 0 0;
+      opacity: .38;
+    }
+
+    &:hover, &:focus {
+      &:before {
+        background: $mainBrandColor;
+        -webkit-mask-image: url('../img/icon-arrow-back.svg');
```

Comment:
Low support for  `mask-image` property. For example in IE we will have wrong hover state.

![image](https://user-images.githubusercontent.com/17984255/46737652-bbaa4080-cca4-11e8-9a4b-6d692b356ab3.png)

Reply:
Please use [border technique](https://www.w3schools.com/howto/howto_css_arrows.asp) or some another.

Reply:
Done